app/Http/Controllers/UserController.php

```diff
@@ -47,8 +51,29 @@ public function followUser(User $profileId)
 
     public function unFollowUser(User $profileId)
     {
-        
         $profileId->followers()->detach(auth()->user()->id);
         return redirect()->back()->with('success', 'Successfully unfollowed the user.');
     }
+
+    public function showCategories()
+    {
+        $categories = Category::all();
+
+        return view('users.Lesson',compact('categories'));
+    }
+
+    public function makeLesson($categoryId)
```

Comment:
I will merge this PR, but i have some comments. this function is not related to Users it is more appropriate that you have a separated controller for Lesson and transfer this function and to that controller